02Lesson/02Project/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css"/>
-</head>
-<body>
+  <head>
+    <title>DAILY PLANET</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/style.css" type="text/css"/>

```

Comment:
You don't need the `/` for self closing tags. `<link ... type="text/css">` would work too.